### PR TITLE
generate avatars for users in seeds.rb

### PR DESCRIPTION
**Pull request by amaseda**
Want to see UI with actual user images.

### CODE REVIEW

**Comment by jshawl** [comment]
:+1:

File: db/seeds.rb [hunk 1]
```diff
@@ -22,7 +22,16 @@ def rand_time( from = Time.now - 4.months, to = Time.now + 4.months)
...
+  case i%4
+    when 0
+      User.create!(name: name, username: username, email: FFaker::Internet.safe_email, password: "foo", image_url: "https://placebear.com/200/200")
+    when 1
+      User.create!(name: name, username: username, email: FFaker::Internet.safe_email, password: "foo", image_url: "http://www.placecage.com/200/200")
+    when 2
+      User.create!(name: name, username: username, email: FFaker::Internet.safe_email, password: "foo", image_url: "http://www.fillmurray.com/200/200")
+    when 3
+      User.create!(name: name, username: username, email: FFaker::Internet.safe_email, password: "foo", image_url: "http://www.stevensegallery.com/200/200")
+  end
```
Comment (adambray) [inline db/seeds.rb]: yassss.... I love it! Can I propose a refactoring?

```
image_urls = [""https://placebear.com/200/200", "http://www.placecage.com/200/200", "http://www.fillmurray.com/200/200", "http://www.stevensegallery.com/200/200"]

User.create!(name: name, username: username, email: FFaker::Internet.safe_email, password: "foo", image_url: image_urls[i%4])
```

Comment (amaseda) [inline db/seeds.rb]: YUP. Implementing now.

**Comment by RobertAKARobin** [comment]
:+1:

**Pull request merged**